Lib/test/test_argparse.py

```diff
@@ -2816,6 +2816,36 @@ def test_interleaved_groups(self):
         result = parser.parse_args('1 2 3 4'.split())
         self.assertEqual(expected, result)
 
+class TestGroupConstructor(TestCase):
+    def test_group_prefix_chars(self):
+        parser = ErrorRaisingArgumentParser()
+
+        msg = (
+            "The use of the undocumented 'prefix_chars' parameter in "
+            "ArgumentParser.add_argument_group is deprecated."
+        )
+        with self.assertWarnsRegex(DeprecationWarning, msg):
+            parser.add_argument_group(prefix_chars='-+')
+
+    def test_group_prefix_chars_default(self):
+        # "default" isn't quite the right word here, but it's the same as
+        # the parser's default prefix so it's a good test
+        parser = ErrorRaisingArgumentParser()
+
+        msg = (
+            "The use of the undocumented 'prefix_chars' parameter in "
+            "ArgumentParser.add_argument_group is deprecated."
+        )
+
+        with self.assertWarnsRegex(DeprecationWarning, msg):
+            parser.add_argument_group(prefix_chars='-')
+
+    def test_group_without_prefix_chars(self):
```

Comment:
This test does not test anything that is not covered by other tests that use `add_argument_group()` without `prefix_chars`.